Sources/MapboxCoreNavigation/EHorizon/RoadGraphPath.swift

```diff
@@ -7,7 +7,7 @@ extension RoadGraph {
     /** A position along a linear object in the road graph. */
     public struct Path {
         /** The edge identifiers that fully or partially coincide with the linear object. */
-        public let edgeIdentifiers: [ElectronicHorizon.Edge.Identifier]
+        public let edgeIdentifiers: [RoadGraph.Edge.Identifier]
```

Comment:
Do we need to explicitly mention `RoadGraph` here(as we are already inside of `RoadGraph`)? Same for `Position`